tools/importer/import.js

```diff
@@ -159,6 +159,20 @@ const createMetadata = (main, document, url) => {
     meta.Image = el;
   }
 
+  // Preference for image alt tag is og:image:alt > og:title > title
```

Comment:
Improve importer to import `imagealt` and set it in doc's metadata block. Subsequently this is indexed and gets used in `article-list` block.